banks-interface/src/lib.rs

```diff
@@ -10,11 +10,19 @@ use solana_sdk::{
     transaction::{self, Transaction, TransactionError},
 };
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum TransactionConfirmationStatus {
+    Unconfirmed,
+    OptimisticallyConfirmed,
+    Finalized,
```

Comment:
How about Max to align with `CommitmentLevel`:
```suggestion
    Max,
```

Reply:
I prefer finalized for clarity, could we change commitment from max to finalized instead since we already planning to modify commitment level?

Reply:
I love that these words are all the same length